pkg/pci/types.go

```diff
@@ -4,8 +4,11 @@
 
 package pci
 
+// Filter can be used to filter a device
+type Filter func(p *PCI) bool
+
 type busReader interface {
```

Comment:
Is there a reason `busReader` is not exported? It feels a bit weird that `NewBusReader()` is returning something not exported. It's also referred to as `BusReader` in the documentation.

Reply:
that's in a later PR but yeah that needs fixing. There's a bunch of little bits to fix in this thing.